CONTRIBUTING.md

```diff
@@ -90,41 +90,43 @@ ID	GLOSS	CONCEPTICON_ID	CONCEPTICON_GLOSS	SIMILARITY
+1 Have you correctly linked your concept list? If so, you should have a linked concept list with minimally the following columns: "ID" (use the form ```FirstAuthorLastName-Year-NumberOfItems-ID``` to create the ID for you concept list), "NUMBER" (a digit, usually following the number in the source, or reflecting the order in the source), "ENGLISH" (if the concept list offers an official English translation) or otherwise "GLOSS" (which requires an additional column indicating the original language of the list), "CONCEPTICON_GLOSS" (the gloss as given in Concepticon), and "CONCEPTICON_ID" (our Concepticon ID).
 2 Have you added the metadata for your concept list? You find more information by looking into the file [```conceptlists.tsv```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/conceptlists.tsv), please contact us if you run in trouble.
-3 Have you added your reference for the concept lists, and all references which you used in your description, to our bibtex file [```references.bib```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/references/references.bib)?
+3 Have you added your reference for the concept list, and all references which you used in your description, to our bibtex file [```references.bib```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/references/references.bib)?
 4 Have you added new concept sets to our Concepticon in the file [```concepticon.tsv```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv)? If so, have you offered proper definitions?
-5 Do you reference a source file in PDF, in your description of the concept list in [```conceptlists.tsv```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/conceptlists.tsv)? If so, have you added it (in the form ```FirstAuthorLastNameYear```) to the folder [```sources/```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/sources/)?
-6 Do you have to thank some person for help with the concept list? If you are contributing and not part of our core editorial team, we will add you, so please provide us with your name and affiliation. To keep track of whom to thank, please edit the [```CONTRIBUTORS.MD```](https://github.com/concepticon/concepticon-data/blob/master/CONTRIBUTORS.md) document.
+5 Do you reference a source file in PDF, in your description of the concept list in [```conceptlists.tsv```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/conceptlists.tsv)? If so, have you added it (in the form ```FirstAuthorLastNameYear```) to the folder [```sources/```](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/sources/)? (AT: Might also need editing.)
```

Comment:
Basically, this is still required and should be done, but recently, we had many lists that were too long (if the source has 500 pages, it is not good to add it), but in many cases, one could still do so or one could HAVE done so, as in the color terms, for example, where a simple screenshot converted to PDF would to the trick.

Basically, I prefer using source files and I created them at times deliberately, even by making many screenshots or the like, as it allows to check and type off data more properly, but I would say: this requirement is weaker now, with the many lists we have added, yet it should still remain here.